manifests/params.pp

```diff
@@ -3,7 +3,7 @@
   $version                    = $globals_version
   $postgis_version            = $globals_postgis_version
   $listen_addresses           = 'localhost'
-  $port                       = 5432
+  $port                       = '5432'
```

Comment:
I do run the parser=future all the time. But since October, some time passed by, maybe that was when I had Puppet 3.7.1 or 3.7.2 running, and where I got an error without the ''. I just tested again, and with and without them, no problem anymore.
I just removed them.

thanks.